CHANGELOG.md

```diff
@@ -3,18 +3,21 @@
 ## v0.2.0 (2019-09-29)
 
 #### :boom: Breaking Change
-* [#11](https://github.com/simplabs/qunit-console-grouper/pull/11) Disable addon for `ember test` ([@Turbo87](https://github.com/Turbo87))
+
+- [#11](https://github.com/simplabs/qunit-console-grouper/pull/11) Disable addon for `ember test` ([@Turbo87](https://github.com/Turbo87))
 
 #### :memo: Documentation
-* [#10](https://github.com/simplabs/qunit-console-grouper/pull/10) README: Add emojis ([@Turbo87](https://github.com/Turbo87))
+
+- [#10](https://github.com/simplabs/qunit-console-grouper/pull/10) README: Add emojis ([@Turbo87](https://github.com/Turbo87))
```

Comment:
what are these changes?

Reply:
These changes are the result of running prettier on the files of the repo, #106's build fails due to [linting errors](https://travis-ci.com/github/simplabs/qunit-console-grouper/builds/164165849).

After the manual upgrade it made sense to run it on the project as prettier v.2 changes some rules, I am following this pr up with a pre-commit hook.

Reply:
- I don't think we should run prettier on anything else than JS in this project
- I don't want a pre-commit hook since it usually doesn't work well with UI clients for git